src/main/java/software/sigma/internship/dto/FundDto.java

```diff
@@ -2,9 +2,12 @@
 
 import javax.validation.constraints.NotBlank;
 import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
 
 public record FundDto(long id,
                       @NotNull @NotBlank String name,
                       @NotNull @NotBlank String description,
+                      @Pattern(regexp = "[(http(s)?):\\/\\/(www\\.)?a-zA-Z0-9@:%._\\+~#=]" +
```

Comment:
Let's create a separate annotation for url validation so that it will be reusable and more maintainable
https://www.baeldung.com/spring-mvc-custom-validator